packages/api/src/platforms/vtex/resolvers/offer.ts

```diff
@@ -105,6 +117,17 @@ export const StoreOffer: Record<string, Resolver<Root>> = {
 
     return null
   },
+  listPriceWithTaxes: (root) => {
+    if (isSearchItem(root)) {
+      return withTax(root.ListPrice, root.Tax, root.product.unitMultiplier)
```

Comment:
question: on `listPrice` it's used `root.ListPrice ?? 0`, isn't the `?? 0` necessary here?

Reply:
To be honest, I just reproduced the current `listPrice` implementation. It looks like the search API could return undefined if the listPrice does not exist, but the order form always returns a value, even if it is 0.